xchange-cexio/src/test/java/org/knowm/xchange/cexio/service/marketdata/RemoteInitTest.java

```diff
@@ -0,0 +1,49 @@
+package org.knowm.xchange.cexio.service.marketdata;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.ExchangeFactory;
+import org.knowm.xchange.cexio.CexIOExchange;
+import org.knowm.xchange.cexio.dto.marketdata.CexIOCurrencyLimits;
+import org.knowm.xchange.cexio.service.CexIOMarketDataService;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author ujjwal on 13/02/18.
+ */
+public class RemoteInitTest {
+  @Test
+  public void unitJsonMapperTest() throws IOException {
+    InputStream is = getClass().getResourceAsStream("/marketdata/sample_currency_limits.json");
+    ObjectMapper mapper = new ObjectMapper();
+    final CexIOCurrencyLimits cexIOCurrencyLimits = mapper.readValue(is, CexIOCurrencyLimits.class);
+    assertThat(cexIOCurrencyLimits).isNotNull();
+    assertThat(cexIOCurrencyLimits.getData().getPairs().size()).isBetween(1, 30);
+    assertThat(cexIOCurrencyLimits.getData().getPairs()).allSatisfy(pair -> {
+      assertThat(pair.getSymbol1()).isNotNull();
+      assertThat(pair.getSymbol2()).isNotNull();
+      assertThat(pair.getMinLotSize()).isPositive();
+      // we don't care about the other parameters for now.
+    });
+
+  }
+
+  @Test
+  public void integrationTest() throws IOException {
+    final Exchange exchange = ExchangeFactory.INSTANCE.createExchange(CexIOExchange.class);
+
+    assertThat(exchange).isNotNull();
+    assertThat(exchange.getExchangeMetaData().getCurrencyPairs()).isNotEmpty();
+
+    final CexIOCurrencyLimits currencyLimits = ((CexIOMarketDataService) exchange.getMarketDataService()).getCurrencyLimits();
```

Comment:
this class makes a remote call to an exchange right? Please rename it to *Integration.java so that it's run as an integration test and not a unit test.

https://github.com/timmolter/XChange/wiki/New-Implementation-Best-Practices#step-9

Reply:
I was wondering if I should separate the unit & Integration tests into separate classes for this reason. What are your thoughts?

Reply:
Addressed.